crates/bitwarden-core/src/mobile/crypto_client.rs

```diff
@@ -39,6 +43,24 @@ impl CryptoClient {
         initialize_org_crypto(&self.client, req).await
     }
 
+    /// Generates a new key pair and encrypts the private key with the provided user key.
+    /// Crypto initialization not required.
+    pub fn make_key_pair(&self, user_key: String) -> Result<MakeKeyPairResponse, CryptoError> {
+        make_key_pair(user_key)
+    }
+
+    /// Verifies a user's asymmetric keys by decrypting the private key with the provided user
+    /// key. Returns if the private key is decryptable and if it is a valid matching key.
+    /// Crypto initialization not required.
+    pub fn verify_asymmetric_keys(
+        &self,
+        request: VerifyAsymmetricKeysRequest,
+    ) -> Result<VerifyAsymmetricKeysResponse, CryptoError> {
+        verify_asymmetric_keys(request)
+    }
+}
+
+impl CryptoClient {
```

Comment:
I've split the `impl CryptoClient` into two blocks, one with wasm_bindgen for the functions we're currently exporting to wasm, and one without it for the ones that we're not using. We could also export all of them if we wanted.